storage/tianmu/core/tianmu_attr.h

```diff
@@ -223,9 +223,34 @@ class TianmuAttr final : public mm::TraceableObject, public PhysicalColumn, publ
   int64_t GetMaxInt64() const { return hdr.max; }
   void SetMaxInt64(int64_t a_imax) { hdr.max = a_imax; }
   bool GetIfAutoInc() const { return ct.GetAutoInc(); }
+  bool GetIfUnsigned() const { return ct.GetUnsigned(); }
   uint64_t AutoIncNext() {
     backup_auto_inc_next_ = hdr.auto_inc_next;
-    return ++hdr.auto_inc_next;
+    if (backup_auto_inc_next_ >= UINT64_MAX)
+      return backup_auto_inc_next_;
```

Comment:
the type of `backup_auto_inc_next_` is unsigend int 64, and its boundary is [UINT64_MIN, UINT64_MAX], and so when the if statment happens.

Reply:
ACK, I will modify it.

Reply:
In this case we only have to deal with the upper boundary， due to  `auto_inc_next >=0`